bin/compile

```diff
@@ -250,6 +250,11 @@ fetch_package "ext/$(php_api_version)/php-redis" "/app/vendor/php" | indent
 echo "       mongodb"
 fetch_package "ext/$(php_api_version)/php-mongodb" "/app/vendor/php" | indent
 
+if [ "$(php_api_version)" = "20170718" -o "$(php_api_version)" = "20180731" ] ; then
+  echo "       mcrypt"
+  fetch_package "ext/$(php_api_version)/php-mcrypt" "/app/vendor/php" | indent
```

Comment:
Please use 4  spaces indentation like the rest of the file.

Reply:
done